source/SkiaSharp.Views.Forms/SkiaSharp.Views.Forms.Native.Shared/SKGLViewRendererBase.cs

```diff
@@ -127,5 +139,22 @@ private void OnPaintSurface(object sender, SKNativePaintGLSurfaceEventArgs e)
 			// the control is being repainted, let the user know
 			controller?.OnPaintSurface(new SKPaintGLSurfaceEventArgs(e.Surface, e.RenderTarget));
 		}
+
+#if __ANDROID__
+		private class Renderer : SKNativeView.ISKRenderer
+		{
+			private readonly ISKGLViewController controller;
+
+			public Renderer(ISKGLViewController controller)
+			{
+				this.controller = controller;
+			}
+
+			public void OnDrawFrame(SKSurface surface, GRBackendRenderTargetDesc renderTarget)
+			{
+				controller.OnPaintSurface(new SKPaintGLSurfaceEventArgs(surface, renderTarget));
+			}
+		}
+#endif
```

Comment:
No wait. My bad.

The Android GL view does not use events. It is the surface-based system, which separates the view from the actual rendering via this render construct.